engine/runtime-language-arrow/src/main/java/org/enso/interpreter/arrow/node/ArrowCastFixedSizeNode.java

```diff
@@ -0,0 +1,16 @@
+package org.enso.interpreter.arrow.node;
+
+import com.oracle.truffle.api.nodes.Node;
+import org.enso.interpreter.arrow.ArrowParser;
+import org.enso.interpreter.arrow.runtime.ArrowCastToFixedSizeArrayFactory;
+
+public class ArrowCastFixedSizeNode extends Node {
+
+  static ArrowCastFixedSizeNode build() {
```

Comment:
Note that the default Truffle convention is to name this method as `create`. If it is not used in any of our DSL, lets rather rename it to `create`.